src/main/java/uk/gov/hmcts/darts/datamanagement/api/impl/DataManagementApiImpl.java

```diff
@@ -79,6 +76,15 @@ public void deleteBlobDataFromUnstructuredContainer(UUID blobId) throws AzureDel
         dataManagementService.deleteBlobData(getUnstructuredContainerName(), blobId);
     }
 
+    @Override
+    public UUID saveBlobDataToInboundContainer(InputStream inputStream) {
```

Comment:
This makes sense. Maybe there is scope for some kind of general upload interface much like we did for the download ability

Reply:
Agree, I'll work on some tickets in this area and will make sure you're involved.